main.py

```diff
@@ -172,6 +184,16 @@ def percent(part, whole):
 		print("Error loading blocker configuration data: ", e)
 		sys.exit()
 
+    #Get set from the list of all bugs in all jobs
+	all_bugs_set = get_bugs_set(blockers)
+	#DEBUG print(list(all_bugs_set))
+
+	#Create dictionary the set of all bugs (key) with name and link as value
+	all_bugs_dict = {}
+	for bug in list(all_bugs_set):
+		all_bugs_dict[bug] = get_bugzilla('',[bug])
+	#DEBUG print(all_bugs_dict)
```

Comment:
```suggestion
	# Create dictionary the set of all bugs (key) with name and link as value
	all_bugs_dict = {}
	for bug in list(all_bugs_set):
		all_bugs_dict[bug] = get_bugzilla('',[bug])
```

Reply:
Maybe we should retool the `get_bugzilla()` function call? It seems as if the `job_name` field is no longer necessary with these changes